bot/interact.go

```diff
@@ -51,8 +51,14 @@ func interactAPIHandler(botContext cmd.BotContext) http.HandlerFunc {
 		case slack.InteractionTypeDialogSubmission:
 			switch message.CallbackID {
 			case "report_message":
+				var user string
+				if message.Submission["anonymous"] == "anonymous" {
+					user = "anonymous"
+				} else {
+					user = message.User.Name
+				}
 				msg := fmt.Sprintf("<@%s> sent a message report:\n- *Reason*: %s\n- *Feeling Scale*: %s of 5\n%s",
```

Comment:
When anonymous, this is gonna look weird since it's adding a link to an account that does not exist in theory: `@anonymous`

Reply:
Fixed!